dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/engine/system/event/GlobalMasterFailoverEvent.java

```diff
@@ -29,7 +29,7 @@ public class GlobalMasterFailoverEvent extends AbstractSystemEvent {
     private final Date eventTime;
 
     public GlobalMasterFailoverEvent(Date eventTime) {
-        super(eventTime.getTime());
+        super(0);
```

Comment:
Please using constant here instead of hard code.

Reply:
done